sdk/typescript/src/types/transactions.ts

```diff
@@ -110,16 +110,19 @@ export type TransactionKindName =
   | 'Genesis';
 
 export const SuiTransactionKind = union([
```

Comment:
Unlike the change in events, I don't see any change on the Explorer/wallet side for transactions, this means 
1. we are using the util functions like `getTransferObjectTransaction` everywhere so that this change is transparent to the wallet and explorer
2. we are missing something, in which case we need to run them locally and manually validate